<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelReviewClass</class>
 <widget class="QWidget" name="LevelReviewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LevelReview</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>661</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="score_label">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#000000</string>
        </property>
        <property name="text">
         <string>  Score :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="score">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:#000000</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,3,3,3">
      <item>
       <widget class="QLabel" name="rating_label">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>36</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">0000ff</string>
        </property>
        <property name="text">
         <string>  Rating :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_3">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_5">
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>65</y>
            <width>54</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PlaneFight.qrc">:/PlaneFight/img/star1.svg</pixmap>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>65</y>
            <width>54</width>
            <height>61</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PlaneFight.qrc">:/PlaneFight/img/star2.svg</pixmap>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_3">
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>54</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PlaneFight.qrc">:/PlaneFight/img/star1.svg</pixmap>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>55</y>
            <width>54</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PlaneFight.qrc">:/PlaneFight/img/star2.svg</pixmap>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>54</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PlaneFight.qrc">:/PlaneFight/img/star1.svg</pixmap>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>60</y>
            <width>54</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="PlaneFight.qrc">:/PlaneFight/img/star2.svg</pixmap>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="Title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>641</width>
     <height>131</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>15</y>
       <width>621</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#ff0000</string>
     </property>
     <property name="text">
      <string>    Mission Completed</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>631</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#0000ff</string>
     </property>
     <property name="text">
      <string>      Mission failed</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Return_bt">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>670</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Return&gt;&gt;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PlaneFight.qrc"/>
 </resources>
 <connections/>
</ui>
